### PR TITLE
combine installation+supported os pages

### LINKED ISSUE #2148: Combine installation page w/ supported OS page
Just what the title says... as a user, I would expect to find information about which OSes Meilisearch supports on the installation page.

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #2148

### CODE REVIEW

**Comment by dichotommy** [comment]
Could you make this a draft, and we'll implement the same changes on the new docs site branch once it has reached a more stable state? Thanks

File: learn/getting_started/installation.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-# Installation
+# Installation and supported operating systems
```
Comment (guimachiavelli) [inline learn/getting_started/installation.md:1]: Is it a problem that the installation guide's sidebar link is super long now?

```suggestion
# Installation and OS support
```

This is shorter, but does it make sense?

Comment (maryamsulemani97) [reply learn/getting_started/installation.md:1]: OS is a well-known acronym, but I think we should use the full form in the title.

**Comment by maryamsulemani97** [comment]
closing for #2274

**Pull request closed without merging**